static/CrossJoin_CommandLogic.py

```diff
@@ -337,29 +342,41 @@ def GetCams(user_input: str):
                         -# This location has {len(x['Views'])} camera(s). 
                         -# Specify a specific camera with `!CrossJoin cams [1-{len(x['Views'])}]`.
                         """),
-                        type = "rich",
-                        timestamp = dt.datetime.now()
+                        type="rich",
+                        timestamp=dt.datetime.now()
                     )
                     main_return.set_image(url = x['Views'][cameraNum]["Url"].replace(" ", "%20") + f"?t={t}")
                     break
                 i += 1
             else:
                 main_return = discord.Embed(
-                    colour = 0x00eaff,
-                    title = f"511 Alberta > {userSearch}",
-                    description = dedent("""
+                    colour=0x00eaff,
+                    title=f"511 Alberta > {userSearch}",
+                    description=dedent("""
                     The search yielded no results.
                     """),
-                    type = "rich",
-                    timestamp = dt.datetime.now()
+                    type="rich",
+                    timestamp=dt.datetime.now()
                 )
         return main_return
     except Exception as e:
         log.info(f"Error: Basic CrossJoin Run Failed. Reason: {e}")
         return Sys.ErrorMessage_Command(str(e))
 
 
-def SendHelp(user_input: str):
+def GetRoadConditions(user_input: str = None):
+    headers = {
+        "content-type": "application/json"
+    }
+    return_text = r.get(
+        "https://511.alberta.ca/api/v2/get/winterroads",
```

Comment:
`https://511.alberta.ca/api/v2/get/winterroads`
should be 
`https://511.alberta.ca/api/v2/get/winterroads?format=json&lang=en`